diff --git a/gtk/app.cc b/gtk/app.cc
index 65f9c0e..5086a7b 100644
--- a/gtk/app.cc
+++ b/gtk/app.cc
@@ -400,7 +400,7 @@ App::TriageError(CdkError error,
       Util::string errorMsg = Util::Format("Error %d: %s - %s\n",
          error, message.c_str(), Util::FormatV(details.c_str(), args).c_str());
       Util::UserWarning(errorMsg.c_str());
-      Log(errorMsg.c_str());
+      Log(errorMsg.c_str(), "%s");
       delete mWindow;
       exit(error);
    }
